examples/embeddable_examples/public/app/presentation_container_example/components/add_button.tsx

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React, { ReactElement, useEffect, useState } from 'react';
+import { EuiButton, EuiContextMenuItem, EuiContextMenuPanel, EuiPopover } from '@elastic/eui';
+import { ADD_PANEL_TRIGGER, UiActionsStart } from '@kbn/ui-actions-plugin/public';
+import { ParentApi } from '../types';
+
+export function AddButton({
+  parentApi,
+  uiActions,
+}: {
+  parentApi: ParentApi;
+  uiActions: UiActionsStart;
+}) {
+  const [isPopoverOpen, setIsPopoverOpen] = useState(false);
+  const [items, setItems] = useState<ReactElement[]>([]);
+
+  useEffect(() => {
+    let cancelled = false;
+
+    const actionContext = { embeddable: parentApi };
+    const actionsPromises = uiActions.getTriggerActions(ADD_PANEL_TRIGGER).map(async (action) => {
+      return {
+        isCompatable: await action.isCompatible(actionContext),
+        action,
+      };
+    });
+
+    Promise.all(actionsPromises).then((actions) => {
+      if (cancelled) {
+        return;
+      }
+
+      const nextItems = actions
+        .filter(
+          ({ action, isCompatable }) => isCompatable && action.id !== 'ACTION_CREATE_ESQL_CHART'
+        )
+        .map(({ action }) => {
+          return (
+            <EuiContextMenuItem
+              key={action.id}
+              icon="share"
+              onClick={() => {
+                action.execute(actionContext);
+                setIsPopoverOpen(false);
+              }}
+            >
+              {action.getDisplayName(actionContext)}
```

Comment:
Not sure why, but the Links panel is showing an incompatible API error when added to this example:


https://github.com/user-attachments/assets/58c106af-9bd3-439e-891e-01feadbd5baa

Reply:
I will just exclude links panel from the list. I want to change this example to only be lens panels once lens is converted.

Reply:
Resolved with 3c848e1c624cfa3da586e00249f8f1c56f69019e. Updated "add links panel" action to use same check as links embeddable so compatibility method returns false when parentApi is not compatible.